README.md

```diff
@@ -14,6 +14,19 @@ The other repos involved in ggtracker are:
   used by this codebase
 
 
+### Running with vagrant
+You can either run ggtracker in a vagrant-controlled virtualbox or on your host operating system (described after this section).
+To run with vagrant run the following commands:
+
+ * Make sure you have vagrant+virtualbox installed on your computer
+ * Run `vagrant up` to create the virtual box
+ * Run `vagrant ssh` to ssh into the vagrant box
+ * Run `ssh-keygen` in the vagrant box to generate ssh-keys for github
+ * Upload your public keys to Github
+ * Clone `ggtracker` project to `vagrant` user's home directory in the vagrant box
```

Comment:
It is not necessary to generate ssh-keys in vagrant. The folder containing the Vagrantfile is accessible at /vagrant in the VM.  A few less steps to repeat when creating the virtual box again.